post_tags/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .request import search_post_by_tag
```

Comment:
Pylint will eventually get mad that there are two separate imports from .request so these should be merged